File: single-language-analysis/dataset-files/java/TestSmell002.java
// Original URL: https://github.com/Glacier173/Git/blob/7fcb4dae0f2c2e9a9d5a931a15398d59462f4195/GitTest.java#L25-L34

public class TestSmell002 {
    @Test
    void testInitialize() throws IOException {

        // Call init method
        Git.initialize();

        // Checks if index  and objects were created
        assertTrue(new File("index").isFile());
        assertTrue(new File("objects").isDirectory());
    }
}

File: single-language-analysis/dataset-files/CSharp/CSharp.Tests/Tests/TestSmell002.cs
using Xunit;
using System.IO;

public class TestSmell002
{
    [Fact]
    public void TestInitialize()
    {
        // Call init method
        Git.Initialize();

        // Checks if index and objects were created
        Assert.True(new FileInfo("index").Exists);
        Assert.True(Directory.Exists("objects"));
    }
}